packages/devextreme-react/src/core/__tests__/props-updating.test.tsx

```diff
@@ -676,6 +676,46 @@ describe('cfg-component option control', () => {
     expect(Widget.option.mock.calls[1]).toEqual(['nestedOption.b', 'const']);
   });
 
+  it('invokes option change guard handlers', () => {
```

Comment:
This test is okay, just please explain what is different in this test compared to this one:

https://github.com/DevExpress/DevExtreme/blob/24_1/packages/devextreme-react/src/core/__tests__/props-updating.test.tsx#L607

Looks like it is almost the same, this test just calls a re-render, which, considering that we call `jest.runAllTimers`, should not make a difference.

Reply:
Yes, this test is similar to the test i've done. I'll remove mine out.